app/Http/Controllers/Controller.php

```
@@ -98,58 +98,81 @@ public function forgot_password(Request $request)
+        $payload = explode('@@@', $decrypted, 3);
+        if (count($payload) === 3) {
+            [$email, $invitationUuid, $password] = $payload;
+        } elseif (count($payload) === 2) {
+            [$email, $password] = $payload;
+            $invitationUuid = null;
+        } else {
+            return null;
+        }
+
+        $email = Str::lower($email);
+        $user = User::whereEmail($email)->first();
```

PHPStan (2.2.7)

[warning] Call to an undefined static method App\Models\User::whereEmail().


(staticMethod.notFound)